Header header
string exercise_id
float32 t_from
float32 t_to
int16 repetitions
bool start
